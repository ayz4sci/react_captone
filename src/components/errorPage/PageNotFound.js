import React from 'react';

const PageNotFound = () =>{
    return(
        <div>
            <div className="alert alert-danger" role="alert">Page Not Found!</div>
        </div>
    );
};

export default PageNotFound;