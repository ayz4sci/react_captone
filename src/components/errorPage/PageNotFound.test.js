import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from './PageNotFound';

describe ('PageNotFound Test', () => {
    it('renders correctly using snapshot', () => {
        const tree = renderer.create(<PageNotFound />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});