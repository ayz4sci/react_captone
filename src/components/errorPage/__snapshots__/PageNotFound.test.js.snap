// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Test renders correctly using snapshot 1`] = `
<div>
  <div
    className="alert alert-danger"
    role="alert"
  >
    Page Not Found!
  </div>
</div>
`;
